.gitignore

```diff
@@ -22,3 +22,6 @@ build/
 test.py
 a.py
 vgcore*
+
+# Other Visual Studio stuff.
```

Comment:
I use a local only Visual Studio project folder and don't want to accidentally commit those files so I ignored them here.